test/integration/harvest/IdleHarvestStrategy_DAI.js

```diff
@@ -100,23 +101,32 @@ describe.only("Idle Harvest Strategy (DAI)", async () => {
 
   it("Deposit", async () => {
     console.log("Idle Harvest Strategy", IdleHarvestStrategy.address);
+    let balanceBefore = await rewardPoolContract.balanceOf(IdleHarvestStrategy.address);
     await DAI.connect(user).approve(IdleHarvestStrategy.address, AMOUNT_TO_TRANSFER);
     await IdleHarvestStrategy.connect(user).deposit(AMOUNT_TO_TRANSFER);
+    let balanceAfter = await rewardPoolContract.balanceOf(IdleHarvestStrategy.address);
```

Comment:
Can you also add an assertion also for checking the increase of strategyTokens `user` (and the opposite in the redeem)